From 79d786e4b3218c99d4bf9bba216dcc4cc522fedb Mon Sep 17 00:00:00 2001
From: thanhbui <thanh.bui.fz@renesas.com>
Date: Tue, 28 Jul 2020 15:37:42 +0700
Subject: [PATCH 430/504] watchdog: renesas-wdt: Register a restart handler for
 R8A7742

Register a restart handler for R8A7742 (RG/G1H) SoC with Cortex-A15
CPU cores.

Signed-off-by: thanhbui <thanh.bui.fz@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/watchdog/renesas_wdt.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/watchdog/renesas_wdt.c b/drivers/watchdog/renesas_wdt.c
index e3a61f3..3782e8f 100644
--- a/drivers/watchdog/renesas_wdt.c
+++ b/drivers/watchdog/renesas_wdt.c
@@ -281,7 +281,8 @@ static int rwdt_probe(struct platform_device *pdev)
 	 * same ARM core architecture (e.g ARM cortex A15) can use same handler
 	 */
 	if (of_machine_is_compatible("renesas,r8a7743") ||
-	    of_machine_is_compatible("renesas,r8a7744")) {
+	    of_machine_is_compatible("renesas,r8a7744") ||
+	    of_machine_is_compatible("renesas,r8a7742")) {
 		priv->restart_handler.notifier_call = rwdt_restart_handler_ca15;
 		/* 255: Highest priority restart handler */
 		priv->restart_handler.priority = 255;
-- 
2.7.4

